internal/api/mfa.go

```diff
@@ -61,6 +69,74 @@ const (
 	QRCodeGenerationErrorMessage   = "Error generating QR Code"
 )
 
+func (a *API) enrollSMSFactor(w http.ResponseWriter, r *http.Request, user *models.User, session *models.Session, params *EnrollFactorParams) error {
```

Comment:
if you pass the request down, you don't need to pass user and session down because it's already stored in the request context.

Reply:
the intention was to avoid fetching the user and session again, but I on further thought I think the overhead should not be substantial